app/views/layouts/application.html.erb

```diff
@@ -9,10 +9,12 @@
   </head>
 
   <body>
+    <%= render partial: 'layouts/nav' %>
     <section class="main">
       <%= render partial: 'shared/errors' %>
       <%= render partial: 'shared/messages' %>
       <%= yield %>
     </section>
+    <%= render partial: 'layouts/footer' %>
   </body>
```

Comment:
I actually don't think we need a footer at all. If we implement infinite scroll correctly, this will just keep going forever and ever and ever and ever and ever and ever ...